cmd/link_package.go

```diff
@@ -21,7 +21,7 @@ func LinkPackage(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interr
 	here := "../.." + strings.Repeat("/..", depth)
 
 	gopath := Gopath()
-	if len(gopath) == 0 {
+	if gopath == "" {
```

Comment:
I'm curious as to why you changed the `len(gopath)` to a string comparison. Since even an empty string allocates a slice, I would think that the `len()` comparison would be faster. Of course, the compiler probably optimizes away any difference anyway.

I certainly wouldn't disapprove of the patch because of this, but I am curious why you'd make this change.